### DISCUSSION TITLE
Online editor support

### DISCUSSION THREAD

**Discussion by JDMCreator** [lvjr/tabularray#96] [Show and tell]
This is to inform you that [latex-tables.com](https://latex-tables.com) now supports importing tables from `tabularray` and aims to support generating tables with `tblr` and `longtblr` environments next year. This is a major development for the project, which you can follow the progress on [this discussion](https://github.com/JDMCreator/LaTeXTableEditor/discussions/49).

**Comment by lvjr** [comment]
Great! Thank you for your nice tool!
